configure-ironic.sh

```diff
@@ -2,6 +2,12 @@
 
 . /bin/ironic-common.sh
 
+USE_HTTP_BASIC=${USE_HTTP_BASIC:-false}
+IRONIC_HTTP_BASIC_USERNAME=${IRONIC_HTTP_BASIC_USERNAME:-"change_me"}
+IRONIC_HTTP_BASIC_PASSWORD=${IRONIC_HTTP_BASIC_PASSWORD:-"change_me"}
+JSONRPC_HTTP_BASIC_USERNAME=$(uuidgen)
+JSONRPC_HTTP_BASIC_PASSWORD=$(uuidgen)
```

Comment:
I haven't tried this but it looks to me like these two values would be generated differently between the ironic-api and conductor, wouldn't they need to match? If its working I guess its because we're starting the api after the conductor (which resets /shared/htpasswd-json-rpc to the generated client values)

Reply:
After the discussion let's use the `IRONIC_*` credentials to setup everything